### PR TITLE
fix: ViewCommand and IngredientTest issues

### PR DESCRIPTION
**Pull request by gigituang**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ViewCommand.java [hunk 1]
```diff
@@ -137,8 +137,8 @@ private Recipe getRecipe(List<Recipe> lastShownList, Index recipeIndex) throws C
     private String getIngredients(Recipe recipe) {
         StringBuilder ingredients = new StringBuilder();
         for (Ingredient ingredient : recipe.getIngredients()) {
-            ingredients.append(String.format("%s %s %s\n", ingredient.getIngredientName(),
-                    ingredient.getQuantity(), ingredient.getQuantifier()));
+            ingredients.append(String.format("%s %s %s\n", ingredient.ingredientName,
+                    ingredient.quantity, ingredient.quantifier));
```
Comment (yusufaine) [inline src/main/java/seedu/address/logic/commands/ViewCommand.java:141]: Minor suggestion:

```suggestion
            ingredients.append(String.format("%s\n", ingredient.toString()));
```

**Review by yusufaine — APPROVED** [review]
Left a minor suggestion for how the ingredients would be appended, functionally the same but this just looks _cleaner_.

**Review by aidanaden — APPROVED** [review]
LGTM! 🍝 🍝 🍝

**Review by blaketingyu — APPROVED** [review]
LGTM 🌮

**Review by yusufaine — CHANGES_REQUESTED** [review]
As mentioned by @Oddcorner changes to RecipeDisplay.java needs to be made as well as.

**Review by yusufaine — APPROVED** [review]

**Pull request merged**